themes/default/layouts/product/esc.html

```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
         </div>
         <ul class="list-none p-0 mx-5 xl:ml-12" data-faq-type="faq-pricing">
             <li class="accordion-item text-2xl py-3">
-                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configurations" "large_header" true) }}
+                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configuration" "large_header" true) }}
```

Comment:
Should always be either "Pulumi ESC" or "Pulumi ESC (and the words)":

```suggestion
                {{ partial "accordian-header" (dict "text" "What is Pulumi ESC (Environments, Secrets and Configuration)?" "large_header" true) }}
```